integration-tests/tests/api/policy/policy-check.spec.ts

```diff
@@ -323,13 +323,12 @@ describe('Schema policy checks', () => {
         '',
         '   - Deprecation reason is required for field foo in type Query. (source: policy-require-deprecation-reason)',
         '',
-        'ℹ Detected 4 changes',
+        'ℹ Detected 3 changes',
         '',
         '   Safe changes:',
         '   - Type User was added',
         '   - Field user was added to object type Query',
         '   - Field Query.foo is deprecated',
-        '   - Directive deprecated was added to field Query.foo',
```

Comment:
Similarly to new types excluding their new fields on changes -- a new deprecation now excludes this redundant change.